docs/installing_ibm_cloud_pak_for_data_on_a_fyre-based_openshift_cluster.md

```diff
@@ -0,0 +1,53 @@
+# Installing IBM Cloud Pak for Data on a FYRE-based OpenShift cluster
+
+## Required Credentials
+
+- [IBM Cloud Pak for Data entitlement key](https://myibm.ibm.com/products-services/containerlibrary)
+- IBM GitHub API key
+- Password for `kubeadmin` user of OpenShift cluster (see e-mail from fyre@us.ibm.com: "[FYRE] OpenShift cluster finished building")
+
+## Linux/macOS/Windows:
+
+- Create FYRE cluster:
+
```

Comment:
Maybe we can add a comment here: "The OpenShift cluster requires at least 220GB of memory which has to be set for your account by a Fyre admin."